### PR TITLE
Added Eclipse Plugin Instructions

### PR DESCRIPTION
**Pull request by priyyanshi**
Added instructions to install the Eclipse plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,22 @@ This project contains:
...
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+# Table of Contents  
+- [Getting Started](##Getting-Started)  
+    - [Prerequisites](###Prerequisites)
+    - [Setup Instructions](###Setup-Instructions)
+- [Eclipse Plugin Installation Instuctions](##Eclipse-Plugin-Installation-Instuctions)
+- [Common Errors](##Common-Errors)
+- [Contributing](##Contributing)
+- [Feedback](##Feedback)
```
Comment (kathrynkodama) [inline README.md:18]: To link to other sections in the same document, only one `#` is needed. ie. `[Feedback](#Feedback)`

Comment (kathrynkodama) [reply README.md:18]: Consider reorganizing the document as follows
```
# Table of Contents  
- [Getting Started](#Getting-Started)  
    -  in this section, put the new instructions on how to install the latest jar through the dropins directory
- [Building](#Building)
    -  in this section, we can link to a separate "Building" document (create under `docs/Building.md`). In this document we can put the Prequisites, Setup Instructions, and Common Errors sections.
- [Contributing](#Contributing)
- [Feedback](#Feedback)
```
What do you think?

**Comment by kathrynkodama** [comment]
@priyyanshi  You will have to sign the Eclipse Contributor Agreement as well. To do so, follow [this link](https://accounts.eclipse.org/legal/eca/validation/75088). Ensure the email you sign up with Eclipse is the same one attached to your GH.

File: README.md [hunk 2]
```diff
@@ -8,13 +8,50 @@ This project contains:
...
+- [Building](#Building)
+    - [Prerequisites](#Prerequisites)
+    - [Setup Instructions](#Setup-Instructions)
+    - [Common Errors](#Common-Errors)
+- [Contributing](#Contributing)
+- [Feedback](#Feedback)
+---
 ## Getting Started
 
+### Eclipse Plugin Installation Instuctions
```
Comment (kathrynkodama) [inline README.md:23]: We can remove this header and instead add a sentence indicating that the following instructions are for installing the plugin  to an Eclipse instance

File: README.md [hunk 3]
```diff
@@ -8,13 +8,50 @@ This project contains:
...
+        - Move the `*.jar` inside the **dropins** folder. 
+    - **For Linux:**
+        - In the terminal, use the following command to locate the directory with the **eclipse.ini** file: `ls -l /usr/bin | grep 'eclipse'`
+        - In this directory, update the **eclipse.ini** file by adding `-clean` to the top of this file, save, and exit the file.
+        - In the same directory, click on **dropins** folder.
+        - Move the `*.jar` inside the **dropins** folder. 
+
+3. Restart the **Eclipse** application.
+
+## Building
```
Comment (kathrynkodama) [inline README.md:48]: I was originally thinking that the building section could be in a separate document altogether. Perhaps under `docs/Building.md`. What do you think?

File: README.md [hunk 4]
```diff
@@ -8,41 +8,49 @@ This project contains:
 - [lsp4jakarta](/lsp4jakarta) - Language Server for Jakarta EE
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+# Table of Contents  
+- [Getting Started](#Getting-Started)  
+- [Building](#Building)
+    - [Prerequisites](#Prerequisites)
+    - [Setup Instructions](#Setup-Instructions)
+    - [Common Errors](#Common-Errors)
```
Comment (kathrynkodama) [inline README.md:16]: Remove these sections from the Table of Contents

Comment (priyyanshi) [reply README.md:16]: Opps, forgot about that!

**Review by kathrynkodama — APPROVED** [review]
Thanks @priyyanshi! Just pending the ECA

**Pull request merged**